setup.py

```diff
@@ -1,46 +1,35 @@
 #!/usr/bin/env python
 # -*- encoding: utf-8 -*-
-from __future__ import absolute_import
+"""Setup routine for OMMBV.
 
-import os
-from setuptools import setup, find_packages
+Note
+----
+package metadata stored in setup.cfg
+
+"""
 
-here = os.path.abspath(os.path.dirname(__file__))
-version_filename = os.path.join('OMMBV', 'version.txt')
-with open(os.path.join(here, version_filename)) as version_file:
-    version = version_file.read().strip()
+import os
+from setuptools import setup
 
 # Include extensions only when not on readthedocs.org
 if os.environ.get('READTHEDOCS', None) == 'True':
     extensions = []
 else:
-    from numpy.distutils.core import setup, Extension
+    from numpy.distutils.core import Extension
+    from numpy.distutils.core import setup  # noqa: F811
```

Comment:
```suggestion
    from numpy.distutils.core import setup
```

Reply:
flake8 or whatever complains about a redefinition of setup if the noqa isn't there. If I don't have the original setup import though then `python setup.py develop` doesn't work.